feat(config): add support for restrictionKeys (#2602)
AmadeusITGroup/otter: docs/linter/eslint-plugin/rules/o3r-restriction-key-tags.md

# @o3r/o3r-restriction-key-tags

> Ensures that `@o3rRestrictionKey` are used with correct value.

Reviewer: ```suggestion
Ensures that the tags `@o3rRestrictionKey` are used with correct values.
```

> **Warning** This rule must be configured to be used.

Reviewer: ```suggestion
> [!WARNING]
> This rule must be configured to be used.
```

## How to use

```json
{

>   "@o3r/o3r-widget-tags": [

Reviewer: ```suggestion
  "@o3r/o3r-restriction-key-tags": [
```

    "error",

>     {

Reviewer 2: copy pasta :)
